### DISCUSSION TITLE
Extension issue in Theia IDE

### DISCUSSION THREAD

**Discussion by Gopikrishna-GitHB** [eclipse-theia/theia-cloud#131] [Q&A] [answered]
When I try to install any extensions which supports browser in Theia IDE web is not working. Attached screenshot for the reference. Here for Example installed Thunder client extension.
![MicrosoftTeams-image (1)](https://user-images.githubusercontent.com/85285581/208909335-a660244c-01f7-4919-ad82-f9e104b01363.png)

**Comment by jfaltermeier** [comment, marked as answer]
Discussion is continued here: https://github.com/eclipse-theia/theia/discussions/11998#discussioncomment-4468988
